packages/compass-import-export/src/utils/csv-header.ts

```diff
@@ -0,0 +1,3 @@
+export function csvHeaderNameToFieldName(name: string) {
+  return name.replace(/\[\d+\]/, '');
```

Comment:
Not a case I’d like to think about, but I guess the point of this project is to handle the complex stuff well … this should probably replace all of the array indices? :grimacing: 

```suggestion
  return name.replace(/\[\d+\]/g, '');
```